platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/BasicConfig.java

```diff
@@ -56,7 +58,8 @@ public record BasicConfig(
         @ConfigProperty(defaultValue = "1000") int jvmPauseReportMs,
         @ConfigProperty(defaultValue = "60s") Duration hangingThreadDuration,
         @ConfigProperty(defaultValue = "data/saved") String emergencyRecoveryFileLoadDir,
-        @ConfigProperty(defaultValue = "0") long genesisFreezeTime) {
+        @ConfigProperty(defaultValue = "0") long genesisFreezeTime,
+        @ConfigProperty(defaultValue = Configuration.EMPTY_LIST) List<Long> nodesToRun) {
```

Comment:
Since you always use this list as `NodeId` you could store these as `List<NodeId>` and create a `ConfigConverter` for `NodeId`

Reply:
Creating config converter for NodeId